src/main/java/net/packet/ByteBufInPacket.java

```diff
@@ -22,6 +22,8 @@ public byte[] getBytes() {
     public byte readByte() {
         return byteBuf.readByte();
     }
+    @Override
+    public short readUnsignedByte() { return byteBuf.readUnsignedByte(); }
```

Comment:
Please add a test, I'd like `ByteBufInPacket` to be fully covered by unit tests.

Reply:
There are already a bunch of tests in `ByteBufInPacketTest.java`, so it should be easy to add.

Reply:
Done! Added 3 unit tests for readUnsignedByte, particularly around the boundaries.